pigeon/notifications.dart

```diff
@@ -94,8 +94,39 @@ class MessagingStyle {
   final bool isGroupConversation;
 }
 
+/// Corresponds to `androidx.core.app.NotificationChannelCompat`
+///
+/// See: https://developer.android.com/reference/androidx/core/app/NotificationChannelCompat
+class NotificationChannel {
```

Comment:
nit: put this at the top of the file, before PendingIntent — that makes a nice logical lifecycle-oriented order

The methods on AndroidNotificationHostApi are already good this way.

The enum class NotificationImportance should get the same treatment, coming before PendingIntentFlag.